### PR TITLE
feat: mentioned user is clickable

### PR DESCRIPTION
**Pull request by evgenusov**

### CODE REVIEW

File: src/utils.tsx [hunk 1]
```diff
@@ -128,9 +128,13 @@ export const parseMarkdown = (
       );
     }
     parsedTextArray.push(
-      <Text style={mentionStyle} key={getKeyComponent()}>{`@${decodeURI(
-        mention.name
-      )}`}</Text>
+      <TouchableWithoutFeedback
+        onPress={() => (handleURL ? handleURL(mention.id) : null)}
+      >
+        <Text style={mentionStyle} key={getKeyComponent()}>{`@${decodeURI(
```
Comment (max-prokopenko) [inline src/utils.tsx:134]: Are you sure that passing function this way won’t generate new object each render?

Comment (evgenusov) [reply src/utils.tsx:134]: It's not a JSX component, so we can't use hooks here

Comment (evgenusov) [reply src/utils.tsx:134]: @max-prokopenko or you mean something else?

Comment (max-prokopenko) [reply src/utils.tsx:134]: I mean if key is like this key={getKeyComponent()} , so would it rerender each time ?

**Pull request merged**